### ISSUE TITLE
Implement network training.

### ISSUE THREAD

**Issue by greyshaman** [greyshaman/runen#20]
**Problem:** In order to process the data, it is necessary to create an appropriate network structure. This network should be able to do so based on the training data.

**Possible solution:** We can use the input data as a reference and create a processing path that will output the required information. After successfully receiving the data, we can save the configuration of the network. However, network optimization has not yet been performed, and this will be a next step.

**Comment by greyshaman** [comment]
Try to use: 

- the gradient descent method (GD, SGD)
- regression to search data model with 
- lasso-regression
- momentum, nesterov momentum
